engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/atom/Atom.java

```diff
@@ -132,17 +135,26 @@ public boolean hasMembers() {
   @CompilerDirectives.TruffleBoundary
   public Array getMembers(boolean includeInternal) {
     Map<String, Function> members = constructor.getDefinitionScope().getMethods().get(constructor.getType());
-    if (members == null) {
-      return Array.allocate(0);
+    Set<String> allMembers = new HashSet<>();
+    if (members != null) {
+      allMembers.addAll(members.keySet());
+    }
+    members = constructor.getType().getDefinitionScope().getMethods().get(constructor.getType());
```

Comment:
Membes are a union of what is in `constructor.getDefinitionScope()` and `constructor.getType().getDefinitionScope()` - ha: what's the difference? Why there is a difference?

Reply:
Consider `Forbidden_Operation.Error`:
The former has `operation` field (and any builtins associated with the type). The latter (now) has `to_display_text` that is the method of `Forbidden_Operation`.

Reply:
I don't see an explicit test for this behavior. Adding a test proving the above via `org.graalvm.polyglot.Value` might be beneficial.

Reply:
That is an indirect, integration test. I was seeking for a unit test that would check 

> The former has `xyz` field. The latter (now) has `abc` that is the method of the type - both are now members

Something clear & simple (not hundreds of cryptic failures across the codebase) that would help me understand what is the desired behavior.

Reply:
done